scripts/write_to_db.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import numpy as np
+from spreadsheets_helper import read_spreadsheet
+from database_helper import connect_to_db, update_row_orm, close_connection
+
+def main():
+    SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
+    SPREADSHEET_ID_INPUT = '1RZDZTswZxI-smwnyi8Oh5ck092hgl_CBxjtQURrczBs'
+    SPREADSHEET_RANGE ='A1:Z10'
```

Comment:
why just 10?

Reply:
since this based on length which can vary, i would prefer if this was passed as an arg to the script as it's a variable